src/microbe_stage/editor/action_data/OrganelleRemoveActionData.cs

```diff
@@ -50,6 +50,7 @@ protected override CombinableActionData CreateDerivedMoveAction(HexPlacementActi
 
     protected override CombinableActionData CreateDerivedRemoveAction(HexMoveActionData<OrganelleTemplate> data)
     {
-        return new OrganelleRemoveActionData(RemovedHex, data.OldLocation, data.OldRotation);
+        return new OrganelleRemoveActionData(RemovedHex, data.OldLocation, data.OldRotation)
+            { GotReplaced = GotReplaced };
```

Comment:
Should the placement and move actions also be updated to preserve the list of replaced things?

Reply:
This property is a bool, indicating whether the remove action is because that something replaces it (so the remove action has 0 cost). Placement and move action don't have this property.

Reply:
Okay, I see, I was confusing this variable with the list of replaced cytoplasms that exists in the placement action.